pullcheck.py

```diff
@@ -65,19 +65,19 @@ def send_check_result_pull(pull, comment):
 		r = requests.post(pull['url'] + '/comments', headers=prepare_headers(), data=json.dumps(prepare_body_pull(pull, comment)).encode("utf8"))

```

Comment:
VERIFICATION
Your Pull Request title: PULLCHECK-1022 Update pullcheck.py

Check result:

VERIFICATION

* Pull Request title must start with prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']
VERIFICATION

* Commit title must start with prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
* Commit title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']